### ISSUE TITLE
Configuring IEEE?

### ISSUE THREAD

**Issue by evelant** [kardia-as/nrf-zboss-ncp#5]
Is there any way to set the coordinate IEEE address so we can keep the same one when migrating from another radio in home assistant? I looked at the nordic docs here but I have little experience with embedded programming so I wasn't able to figure it out. https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/protocols/zigbee/configuring.html#ug-zigbee-configuring-eui64

**Comment by DamKast** [comment]
Yes, it is possible using the ZBOSS NCP protocol. If you look in the ZBOSS NCP protocol specification or in the zigpy-zboss implementation, you'll find the command called `SET_LOCAL_IEEE_ADDR` (see [zigpy-zboss](https://github.com/kardia-as/zigpy-zboss))
